.github/workflows/ci.yml

```
@@ -46,6 +46,39 @@ jobs:
+    # Packs LevelUp.Strategos.Agents at the pinned v2.7.0-preview.2
+    # version into a directory-based local feed, then restores + builds
+    # + runs the smoke project at tests/basileus-smoke/. The smoke
+    # project's BasileusConsumedSurfaceTests asserts (a) the basileus-
+    # consumed public surface (IConversationThreadManager,
+    # IWorkflowAgentFactory, IStreamingHandler) still compiles against
+    # a mock implementation and (b) the post-DR-11 deletion of the
+    # single-arity IAgentStep<>/AgentStepBase<> types is reflected in
+    # the packed artifact (not just the in-repo build).
+    name: Basileus smoke (release-readiness)
+    needs: build-test
+    runs-on: blacksmith-4vcpu-ubuntu-2204
```

actionlint (1.7.12)

[error] label "blacksmith-4vcpu-ubuntu-2204" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)